### PR TITLE
Add link

### PR DESCRIPTION
**Pull request by andreyradkevich**
In this piece of code, we created opportunity add link into DB

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
+// this module is intended for routing
 var express = require('express');
 var path = require('path');
 var favicon = require('serve-favicon');
 var logger = require('morgan');
 var cookieParser = require('cookie-parser');
+//You need to use bodyParser() if you want the form data to be available in req.body.
 var bodyParser = require('body-parser');
-
+const SECRET_KEY = "asdfsfsdfs";
```
Comment (zavodnyuk) [inline app.js]: move to configs, and .gitignore
it's not secret any more:)

File: config/database.js [hunk 2]
```diff
@@ -2,5 +2,5 @@
  * Created by Andrey Radkevich on 03.05.2017.
  */
 module.exports = {
-    url: 'mongodb://185.86.76.179/27017',
+    url: 'mongodb://localhost:27017',
```
Comment (zavodnyuk) [inline config/database.js]: the url can also be in configs and invisible for everyone, just in local

File: config/signup_test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/**
```
Comment (zavodnyuk) [inline config/signup_test.js]: rename to signup.test.js

File: routes/addLinkToDB/addLinkToDB.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by Andrey Radkevich on 13.06.2017.
+ */
+
+
+
```
Comment (zavodnyuk) [inline routes/addLinkToDB/addLinkToDB.js]: do you need this function?

File: api/links/addToDb_test.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            .post ('/torrent-link/add')
+            .send( {
+                link : 'http://rutor.co/3032qqwe-chuzhoyfdsfsd-zavet-2017.html'
+            })
+            .expect(200)
+            .end(function (err, res) {
+                if (err) {
+                    return done(err);
+                }
+                console.log(res.body)
```
Comment (zavodnyuk) [inline api/links/addToDb_test.js]: tests not finished..you need to compare data you sent with data you received like **res.body.link.should.be.equal(link)**
console.log it's just for seeing if something came back. but this tests actually do nothing

**Comment by andreyradkevich** [comment]
Check please

**Pull request closed without merging**